test/detect.js

```diff
@@ -31,7 +31,7 @@ test('Should ignore if matching any framework.excludedNpmDependencies', async (t
 
 test('Should detect config files', async (t) => {
   const frameworks = await getFrameworks('config_files')
-  t.is(frameworks.length, 1)
+  t.is(frameworks.length, 2)
```

Comment:
Because `vite` exists within the project, `framework-info` is now picking that up as well as the framework for which these were originally checking for

Reply:
Adding a package.json in the `test/fixtures/config_files` folder like in `test/fixtures/empty` fixes this, so it only picks up the one framework, `middleman`.

e.g. package.json
```
{
  "name": "test",
  "version": "1.0.0"
}
```

So this can stay as

```suggestion
  t.is(frameworks.length, 1)
```